tenseal/tensors/ckksvector.cpp

```diff
@@ -206,6 +206,20 @@ CKKSVector CKKSVector::sub_plain(double to_sub) {
     return new_vector;
 }
 
+CKKSVector CKKSVector::rsub_plain(vector<double> left_operand) {
```

Comment:
this create an unnecessary copy of the vector.

Reply:
but we can't do an inplace operation, or how can we prevent the copy?

Reply:
`this->negate()` will return a new vector, and the following `add_plain_inplace` won't create a new vector.
we have to create a new vector, because we can't modify the vector when it's on the right side of the operation.
example: 
v  = [1,2,3] # ckks encrypted vector
x = 2 - v  # scaler operation 
x == [1, 0, -1]
in this case v shouldn't change.

Reply:
I was referring that left_operand should be const vector<double>&, now it is copied to the parameter.

Reply:
yeah, you are write, we need to fix that in all functions